index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Project Day 2</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+    <header class="header">
+      <nav class="nav">
+	<a href="#" class="logo">      
+	  <h3>SJA</h3>
+        <img class="btn" src="img/Union.png" alt="Union"/>
+	</a>	
+      </nav>
+    </header>
+
+
+  <section class="about">
+          <h1>SJA</h1>
+          <p class="para">Software Developer</p>
+          <p class="par">I can help you build a product ,<br /> feature or website. Look through <br/>some of my work and experience!<br> If you like what you see and have a <br>project you need coded, <br> don’t hestiate to contact me.
+          </p>
+</section>
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/133516381-3824a600-1b7d-404f-af89-d264d4ebcadd.png)

- [ ] As the previous reviewer suggested. You should use the same font family and colors used in the design. check the Figma template for style information. Respecting the original design is one of the project's requirements as it's an important skill for a front-end developer. 
original design :
![image](https://user-images.githubusercontent.com/43698511/133516035-4b175b16-f286-4a61-944f-cbac162e2ab3.png)
You can use this link to lookup [the font](https://fonts.google.com/)